app/models/card.rb

```diff
@@ -3,6 +3,14 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :add_review_date, presence: true
   validate :words_cannot_be_equal
 
+  scope :random_for_review, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.